contracts/src/utils/TokenScalingUtils.sol

```diff
@@ -34,6 +36,19 @@ library TokenScalingUtils {
         return _scaleTokens(tokenMultiplier, multiplyOnDestination, destinationTokenAmount, false);
     }
 
+    function deriveFactors(
```

Comment:
Add a natspec description of this function.

Reply:
addressed in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/b27d7ac63a4064f98acae7823ea4ac7f0e1b3a65